src/main/webapp/resources/js/components/icons/Success.jsx

```diff
@@ -16,6 +16,9 @@ import { SPACE_SM } from "../../styles/spacing";
 
 export function Success() {
```

Comment:
I was thinking not of an Icon exactly, but more of a component that you could pass in a message so you could see both the icon and the message.

Reply:
Isn't that what we did for the Alerts?